source/project_description.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="project-description">
+  <p>
+    Refills are intended to be used as a reference for accessibility, and as a
```

Comment:
I think the following might be preferable:
> Refills are intended to be used as a starting point for building interactive patterns and components that are accessable, responsive, and semantic.

Otherwise it sounds like accessability is the only use case.